src/extension.ts

```diff
@@ -83,7 +88,7 @@ export function activate(context: vscode.ExtensionContext): void {
         const siteType = util.isSiteDeploymentSlot(node.treeItem.site) ? 'Deployment Slot' : 'Web App';
         outputChannel.show();
         outputChannel.appendLine(`Starting ${siteType} "${node.treeItem.site.name}"...`);
-        await node.treeItem.start(nodeUtils.getWebSiteClient(node));
+        await node.treeItem.siteWrapper.start(nodeUtils.getWebSiteClient(node));
```

Comment:
Looks like another merge issue. This should still be node.treeItem.start

Same with the stop command below

Reply:
Actually, this one I changed because I didn't think that the start/stop methods lived in the treeItem itself, but in the siteWrapper.

Reply:
I added start/stop to the treeItem, which calls siteWrapper and updates the status text in node label.